coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -162,14 +162,21 @@ def _extract_doc_comment(content, line, column, markers):
                     line, column and the extracted documentation. If not
                     matched, returns None.
     """
-    if markers[1] == '':
-        # Extract and align to start marker.
-        return _extract_doc_comment_simple(content, line, column, markers)
-    elif markers[1] == markers[2]:
-        # Search for the each-line marker until it runs out.
-        return _extract_doc_comment_continuous(content, line, column, markers)
-    else:
-        return _extract_doc_comment_standard(content, line, column, markers)
+    # This ignores fancy docstrings.
+    valid_docstring_regex = '\s*[' + markers[0] + '].*'
```

Comment:
Further modified regex :tada: got this idea from `begin_regex` no need of group matching `()`. we now add escape sequence before each marker character(This is required as for other languages special character needs escape sequences eg: java `*/**`). So for python the regex is now
`\s*\"|\"|\".*`